db/migrations/20220104165950_goteo_data_sets.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Migration Task class.
+ */
+class GoteoDataSets
+{
+  public function preUp()
+  {
+      // add the pre-migration code here
+  }
+
+  public function postUp()
+  {
+      // add the post-migration code here
+  }
+
+  public function preDown()
+  {
+      // add the pre-migration code here
+  }
+
+  public function postDown()
+  {
+      // add the post-migration code here
+  }
+
+  /**
+   * Return the SQL statements for the Up migration
+   *
+   * @return string The SQL string to execute for the Up migration.
+   */
+  public function getUpSQL()
+  {
+     return "CREATE TABLE `data_set` (
+            `id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+            `title` TEXT NULL,
+            `description` VARCHAR(10) NULL,
```

Comment:
@davidbeig I don't know the details of this task, but 10 characters seem to be too little for a description column.